manifests/m/MattTytel/Helm/0.9.0/MattTytel.Helm.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MattTytel.Helm
+PackageVersion: 0.9.0

```

```suggestion
PackageVersion: 0.9.0.0
```
This must match control panel